### PR TITLE
rocksdb: add get_compression method

### PR DESCRIPTION
**Pull request by choleraehyq**
@zhangjinpeng1987 PTAL

Signed-off-by: Cholerae Hu <huyingqian@pingcap.com>

### CODE REVIEW

**Comment by choleraehyq** [comment]
@zhangjinpeng1987 PTAL

File: tests/test_rocksdb_options.rs [hunk 1]
```diff
@@ -352,3 +353,14 @@ fn test_allow_concurrent_memtable_write() {
         db.put(format!("k_{}", i).as_bytes(), b"v").unwrap();
     }
 }
+
+#[test]
+fn test_get_compression() {
+    let mut opts = Options::new();
+    opts.create_if_missing(true);
+    opts.compression(DBCompressionType::DBSnappy);
+    match opts.get_compression() {
```
Comment (siddontang) [inline tests/test_rocksdb_options.rs]: use assert_eq!

Comment (choleraehyq) [inline tests/test_rocksdb_options.rs]: `DBCompressionType` does't implement `std::cmp::PartialEq` and `std::fmt::Debug`, so cannot use `assert_eq!`.

**Comment by choleraehyq** [comment]
Done. @siddontang

**Comment by andelf** [comment]
LGTM

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**